apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts

```diff
@@ -415,6 +415,162 @@ for (const { swap, xrayTicket } of tooLowAmountForQuoteSwaps) {
   });
 }
 
+const swapEntryPoint = {
+  swap: new Swap(
+    Account.BTC_NATIVE_SEGWIT_1,
+    Account.ETH_1,
+    "0.0006",
+    Fee.MEDIUM,
+    Provider.CHANGELLY,
+    Rate.FLOAT,
+  ),
+};
+
+test.describe("Swap flow from different entry point", () => {
+  test.beforeAll(async () => {
+    process.env.SWAP_DISABLE_APPS_INSTALL = "true";
```

Comment:
yep, I think you can remove the broadcasting var

Reply:
`SWAP_DISABLE_APPS_INSTALL`  is actually to prevent apps version check and rely on those we considered `app candidates` (prod versions) for `speculos`
`SWAP_API_BASE` is to target correct firebase env with proper flags